lib/ts/recipe/multitenancy/recipeImplementation.ts

```diff
@@ -113,5 +113,10 @@ export default function getRecipeInterface(querier: Querier): RecipeInterface {
             );
             return response;
         },
+
+        listAllCoreConfigProperties: async function ({ userContext }) {
+            let response = await querier.sendGetRequest(new NormalisedURLPath(`/core-config/list`), {}, userContext);
```

Comment:
need to pass in recipe id?

Reply:
No this API is not dependent on any recipe, have moved it out of this file